lib/bundler.rb

```diff
@@ -170,6 +170,7 @@ def user_home
         end
 
         if warning
+          Kernel.send(:require, "etc")
```

Comment:
I'm trying to understand why we need to call `.send` here. Would it work if we just called `Kernel.require 'etc'`

Reply:
I was confused by this too. I think `require "etc"` should work just fine (or `Kernel.require 'etc'`) as you suggest.